### PR TITLE
Cut release 2022.7.1

### PR DESCRIPTION
**Pull request by kparwal**
New release.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: athena-aws-cmdb/athena-aws-cmdb.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ Metadata:
     Labels:
       - athena-federation
     HomePageUrl: 'https://github.com/awslabs/aws-athena-query-federation'
-    SemanticVersion: 2022.4.1
+    SemanticVersion: 2022.7.0
```
Comment (janmran) [inline athena-aws-cmdb/athena-aws-cmdb.yaml:13]: 2022.7.1 (iteration will start at 1)

Comment (janmran) [reply athena-aws-cmdb/athena-aws-cmdb.yaml:13]: also need to update the version in pom files. take a look at https://github.com/awslabs/aws-athena-query-federation/pull/665/files (usually just increment the last number)

**Review by janmran — CHANGES_REQUESTED** [review]

File: athena-aws-cmdb/athena-aws-cmdb.yaml (hunk 1 above)
Comment (kparwal) [reply athena-aws-cmdb/athena-aws-cmdb.yaml:13]: will do

Comment (kparwal) [reply athena-aws-cmdb/athena-aws-cmdb.yaml]: fixed

**Review by janmran — APPROVED** [review]

**Pull request merged**